Tests/SentryTests/SentryTests.m

```diff
@@ -21,7 +21,10 @@ - (void)testVersion
 {
     NSDictionary *info = [[NSBundle bundleForClass:[SentryClient class]] infoDictionary];
     NSString *version = [NSString stringWithFormat:@"%@", info[@"CFBundleShortVersionString"]];
-    XCTAssert([version isEqualToString:SentryMeta.versionString]);
+    if ([info[@"CFBundleIdentifier"] isEqualToString:@"io.sentry.Sentry"]) {
+        // Tests running as part of another build
```

Comment:
`m`: I don't fully understand this comment, it is a little bit confusing to me and I see a risk of someone touching this in the future and not knowing what it is about.

Reply:
The goal is to allow the version of the bundle differ from our hard coded version. That's OK in case the bundle identifier isn't equal to `io.sentry.Sentry`. 

Inside the `if`: Tests are running as part of another build (not Sentry build, some customer project or else)